ms2deepscore/SettingsMS2Deepscore.py

```diff
@@ -101,8 +101,9 @@ def __init__(self, **settings):
         self.dropout_rate = 0.2
         self.learning_rate = 0.00025
         self.epochs = 150
-        self.patience = 10
+        self.patience = 15
```

Comment:
I normally use an even higher patience, but this doesn't have to be exactly the same for both of us. The rest looks like good defaults.

Reply:
We could also turn of dropout by default?